docs/DevelopmentSetup/windows_installation.rst

```diff
@@ -42,12 +42,34 @@ cloned earlier) and use the following commands:
 
    Installing required packages using vcpkg in Command Prompt
 
+
+Setting up the FLINT development environment
+--------------------------------------------
+- Clone the FLINT repository in the moja-global folder from `here <https://github.com/moja-global/FLINT>`_
+- After cloning the FLINT repository, switch to ``develop`` branch.
+
+
+.. figure:: ../images/windows_installation/develop_branch.png
+   :alt: Develop branch of FLINT
+   :width: 600
+   :align: center
+
+   Develop Branch of FLINT
+
+-  Start command shell in the FLINT folder of your system and run
+   command:
+
+::
+
+   cd FLINT
+   git checkout develop
+
 Using cmake to build the project
 --------------------------------
 
 .. note::
 
-   Please make sure that you have checked out to the ``Develop`` Branch
+   Please make sure that you have checked out to the ``develop`` Branch
```

Comment:
This statement seems unnecessary as the user must have switched to the `develop` branch already in the previous step.

Reply:
Shall I add it [here](https://docs.moja.global/en/latest/DevelopmentSetup/git_and_github_guide.html#fork-and-clone-this-project)

Reply:
Yes! But it seems unnecessary in the Windows Installation guide. 😅
We do not want to confuse the user by breaking the step by step flow.

Reply:
> Yes! But it seems unnecessary in the Windows Installation guide. 😅
> We do not want to confuse the user by breaking the step by step flow.

Han okay will remove it from the Windows Installation guide!